### ISSUE TITLE
Chrome 111 breakpoints don't work

### ISSUE THREAD

**Issue by connor4312** [microsoft/vscode-js-debug#1604]
Chrome made a change in 111 which causes an error in the debugger. They're [rolling this back](https://bugs.chromium.org/p/chromium/issues/detail?id=1421661) in 112, and we're considering a recovery release in the next few days.

**To fix this, use the nightly debugger build:**

1. Open the extensions view (ctrl+shift+x) and search for @builtin @id:ms-vscode.js-debug
2. Right click on the JavaScript Debugger extension and Disable it.
3. Search for @id:ms-vscode.js-debug-nightly in the extensions view.
4. Install that extension.

**Comment by connor4312** [comment]
We are releasing a recovery build for a fix in VS Code within the next few days.

**Comment by connor4312** [comment]
@joyceerhl verified this

**Comment by Bernoulli-IT** [comment]
Works 👍🏼 phieuw...

**Comment by starball5** [comment]
Related Stack Overflow questions:

- [VSCode debugger blank page and loading on launch with VS Code 1.76.1 and Chrome 111]([VSCode debugger blank page and loading on launch with VS Code 1.76.1 and Chrome 111](https://stackoverflow.com/questions/75723803/vscode-debugger-blank-page-and-loading-on-launch-with-vs-code-1-76-1-and-chrome)) I'd prefer using this one as a duplicate target.
- [Blank Chrome screen with persistant loading spinner when launching the debugger using pwa-chrome in VS Code. How can I debug this?](https://stackoverflow.com/q/75731163/11107541)
- [VSCode / Angular application not fully launching](https://stackoverflow.com/q/75697607/11107541)
- [VScode's debugger stuck after update (Running Angular app)](https://stackoverflow.com/q/75724768/11107541)
- [Why when breakpoint is present in my solution, the browser launched by the Javascript Debugger never actually resolve the page?](https://stackoverflow.com/q/75746312/11107541)
- [What are the steps to debug an inability bind breakpoints on a chrome launch configuration using Angular?](https://stackoverflow.com/q/75741049/11107541)

Help with flagging the others as duplicates of the first would be appreciated.

**Comment by connor4312** [comment]
Thanks, @starball5, I've added my vote to duplicate to those issues. I've also linked your comment on the Chromium issue to lend weight to the suggestion to backport to 111.

**Comment by starball5** [comment]
@connor4312 thanks for the help dup-voting! I'll keep updating that list with new possible duplicates as they come it (for example, I just added one to the bottom)

**Comment by IWSR** [comment]
Thanks you for your help！It works.

**Comment by Amitkad** [comment]
Still an issue when refreshing the page or hot-reloading unless I clear all breakpoints and launch debugger again

**Comment by jirigalis** [comment]
+1, everything worked well until now. I had updated vscode-js-debugger to the nightly version and the debugging process worked. Now the issue is back. Still using the nightly version of debugger, chrome v111, vscode 1.76.2.

**Comment by jirigalis** [comment]
> > +1, everything worked well until now. I had updated vscode-js-debugger to the nightly version and the debugging process worked. Now the issue is back. Still using the nightly version of debugger, chrome v111, vscode 1.76.2.
> 
> @jirigalis sure to have disabled the built-in javascript debugger? Here it "still" works.

I have re-enabled the built-in js debugger and disabled the nightly debugger. Everything seems to be working again now. Thanks a lot!

**Comment by YhongChou** [comment]
upgrading it works!

**Comment by gerardparareda** [comment]
I'm experiencing this with Chrome v114 and VS Code 1.79.2. I'm getting a black screen until I remove the breakpoints.

**Comment by connor4312** [comment]
You're experiencing an unrelated issue, please create a new one.

**Comment by jonas-lomholdt** [comment]
I also have this issue now. Tried using both chrome and edge, but the page is just blank and keeps loading forever. Anyone seeing this issue again?

**Comment by jonas-lomholdt** [comment]
> I'm experiencing this with Chrome 114.0.5735.106 and VS Code 1.79.2. I'm getting a black screen until I remove the breakpoints. So, upgrading does not work _now_.

Ok this helped, if I remove the breakpoints it will start, and setting the breakpoints after it started works. But If I have any breakpoints set before starting it will hang using both edge and chrome.

**Comment by nmathieu-onshape** [comment]
I am also hitting this again, Chrome Version 130.0.6723.117 (Official Build) (arm64) and VSCode Version: 1.95.2
